src/templates/components/facilityListing/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { MediaImage } from '@/templates/common/mediaImage'
+import { OperatingStatusFlags } from '@/templates/layouts/healthCareLocalFacility/OperatingStatus'
+import { PhoneNumber } from '@/templates/common/phoneNumber'
+import { formatter as formatParagraphPhoneNumber } from '@/data/queries/phoneNumber'
+import { Address } from '@/templates/layouts/healthCareLocalFacility/Address'
+import { MinimalLocalFacility } from '@/types/formatted/vamcSystem'
+
+type FacilityListingProps = {
+  // TODO: Figure out what this type should actually be
+  facility: MinimalLocalFacility
```

Comment:
Maybe like this?
```suggestion
  facility: Pick<HealthCareLocalFacility, 'title' | 'path' | 'operatingStatusFacility'> // etc.
```

Reply:
That's fair. I want to be able to reuse a type that is a subset, so maybe `type MinimalLocalFacility = Pick<HealthCareLocalFacility, ...`

Reply:
I forgot about that TODO comment

Reply:
0753e258c87b7fd7f180167ccaf880f3cb7320e3